src/raster/r.maxent.train/r.maxent.train.py

```diff
@@ -0,0 +1,1125 @@
+                )
+                gs.run_command(
+                    "v.db.addcolumn",
+                    map=prediction_layers[0],
+                    columns="tmp999 double precision",
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.pyupdate",
+                    map=prediction_layers[0],
+                    column="tmp999",
+                    expression=vars_max,
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.addcolumn",
+                    map=prediction_layers[0],
+                    columns=f"{nm}_mean double precision",
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.pyupdate",
+                    map=prediction_layers[0],
+                    column=f"{nm}_mean",
+                    expression=vars_mean,
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.addcolumn",
+                    map=prediction_layers[0],
+                    columns=f"{nm}_range double precision",
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.pyupdate",
+                    map=prediction_layers[0],
+                    column=f"{nm}_range",
+                    expression=vars_range,
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.dropcolumn",
+                    map=prediction_layers[0],
+                    columns="tmp888",
+                    quiet=function_verbosity,
+                )
+                gs.run_command(
+                    "v.db.dropcolumn",
+                    map=prediction_layers[0],
+                    columns="tmp999",
+                    quiet=function_verbosity,
+                )
+                for d in vars:
+                    gs.run_command(
+                        "v.db.dropcolumn",
+                        map=prediction_layers[0],
+                        columns=d,
+                        quiet=function_verbosity,
+                    )
+            else:
+                gs.warning(
+                    "Install v.db.pyupdate if you want summary stats\n"
+                    "instead of stats for each submodel"
+                )
+        if bool(options["samplepredictions"]):
+            newname = f"{options['samplepredictions']}{options['suffix']}"
+        else:
+            newname = (
+                prediction_csv[0].replace(".csv", options["suffix"]).replace("_0_", "_")
+            )
+        gs.run_command(
+            "g.rename",
+            vector=f"{prediction_layers[0]},{newname}",
+            quiet=function_verbosity,
+        )
+        gs.info(_("Created the layer {} in grass gis".format(newname)))
```

Comment:
```suggestion
        gs.info(_("Created the layer {} in GRASS GIS".format(newname)))
```

Should grass be GRASS?